sdk/healthdataaiservices/azure-health-deidentification/src/main/java/com/azure/health/deidentification/models/DeidentificationCustomizationOptions.java

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com.azure.health.deidentification.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.annotation.Generated;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+
+/**
+ * Customizations options to override default service behaviors for synchronous usage.
+ */
+@Fluent
+public final class DeidentificationCustomizationOptions
+    implements JsonSerializable<DeidentificationCustomizationOptions> {
+    /*
+     * Format of the redacted output. Only valid when Operation is Redact.
+     * Please refer to https://learn.microsoft.com/en-us/azure/healthcare-apis/deidentification/redaction-format for
+     * more details.
+     */
+    @Generated
+    private String redactionFormat;
+
+    /*
+     * Locale in which the output surrogates are written.
+     */
+    @Generated
+    private String surrogateLocale;
+
+    /**
+     * Creates an instance of DeidentificationCustomizationOptions class.
+     */
+    @Generated
+    public DeidentificationCustomizationOptions() {
+    }
+
+    /**
+     * Get the redactionFormat property: Format of the redacted output. Only valid when Operation is Redact.
+     * Please refer to https://learn.microsoft.com/en-us/azure/healthcare-apis/deidentification/redaction-format for
```

Comment:
Could you double check your typespec about whether they contain the locale `en-us` in `@doc`? Public doc should not have locale that force user get to English web site.

If possible, fix them and regen the SDK.

Reply:
I opened https://github.com/Azure/azure-rest-api-specs/pull/32711 but it's failing on Java SDK generation because the Java TypeSpec options already define a customization file that is being added in this PR. Can you suppress the failure?

Reply:
@weidongxu-microsoft as part of that PR I'm also enabling partial update for our Java SDK since that is an easier path than the customizations that were included in this PR. If it turns out to be better to complete this PR before the TypeSpec PR, it should be fine to switch from customization to partial update as long as the public API surface doesn't change, right?

Reply:
I've approved the specs PR.

Yes, it is fine to enable partial-update. They can actually be used together. For some change it is much easier to do it via partial-update. Though for some changes, customization may be the only option for now.